api/src/main/java/com/epam/pipeline/manager/search/SearchResultConverter.java

```diff
@@ -72,19 +73,27 @@ public SearchResult buildResult(final SearchResponse searchResult,
                 .build();
     }
 
-    public StorageUsage buildStorageUsageResponse(final SearchResponse searchResponse,
+    public StorageUsage buildStorageUsageResponse(final MultiSearchResponse searchResponse,
                                                   final AbstractDataStorage dataStorage, final String path) {
-        final Long size = Optional.ofNullable(searchResponse.getAggregations())
-                .map(aggregations -> aggregations.<ParsedSum>get(STORAGE_SIZE_AGG_NAME))
-                .map(result -> new Double(result.getValue()).longValue())
-                .orElse(0L);
+        final MultiSearchResponse.Item[] responses = searchResponse.getResponses();
```

Comment:
Maybe we need to check whether we have expected number of successful requests and add some NPE & size checks?

Reply:
Done, checking MultiSearchResponse structure and each of the expected nested responses